### PR TITLE
VCP 119: Login Interface

### PR DESCRIPTION
**Pull request by Taysiee**
Redirects admin and user to proper URLs

### CODE REVIEW

File: client/src/components/Login/LoginInterface.js [hunk 1]
```diff
@@ -53,7 +53,34 @@ export class LoginInterface extends Component {
       
     if(rules.required){
       isValid = value.trim() !== '' && isValid;
-    } 
+    }
```
Comment (mackosx) [inline client/src/components/Login/LoginInterface.js]: Do we need validation on login? I'm trying to think of a website that does this, usually its just on signup. So then if requirements for login change, for example if a website increased the minimum length of password, users who signed up before the change would not be able to login.

Comment (Taysiee) [reply client/src/components/Login/LoginInterface.js]: This only checks to see if there is input in both fields so if nothing is inserted then it won't let the user submit. This doesn't need to be implemented but I think this is a nice feature.

File: client/src/components/Login/LoginInterface.js [hunk 2]
```diff
@@ -53,7 +53,34 @@ export class LoginInterface extends Component {
...
+
+    if (rules.minLength) {
+      isValid = value.length >= rules.minLength && isValid;
+      console.log("minlength: " + isValid);
+      isValid === false ? this.setError("password", "Password must be longer than 5 characters") : this.removeError("email");
+    }
+
+    if (rules.isEmail) {
```
Comment (mackosx) [inline client/src/components/Login/LoginInterface.js]: I mean this down here

Comment (mackosx) [reply client/src/components/Login/LoginInterface.js]: Oh I see, it looks like these rules are here but not being used. Should be removed then.

**Pull request merged**